languages/typescript/tree-sitter/Parse_typescript_tree_sitter.ml

```diff
@@ -1758,14 +1758,14 @@ and primary_type (env : env) (x : CST.primary_type) : type_ =
   | `Gene_type x -> TyName (generic_type env x)
   | `Obj_type x ->
       let t1, xs, t2 = object_type env x in
-      let _xs =
+      let xs =
         xs
         |> Common.map_filter (function
              (* TODO *)
```

Comment:
Remove this TODO

Reply:
we still are not dealing with the `Right` case, i'll just move it down one line?

Reply:
Oh I see, sure